examples/gobot_md_yaml_configs_tutorial.ipynb

```diff
@@ -285,6 +296,31 @@
         "**See examples below in this tutorial**"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "efm7WXbbK2L4",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Install the DeepPavlov library\n",
+        "\n",
+        "First things first: we need to install the `deeppavlov` library."
```

Comment:
Either remove `The next step is to install the deeppavlov library` with library installation from branch below (14 cells below) or remove this block and replace installation from github with `pip install deeppavlov` below.

Reply:
agreed with Fedor